data/test_domains/conditional_response_variations.yml

```diff
@@ -12,8 +12,12 @@ slots:
     values:
       - primary
       - secondary
+    mappings:
+      - type: from_text
   can_withdraw:
     type: bool
+    mappings:
+      - type: custom
```

Comment:
Yep, this means that the slot should be set according to a custom mapping defined in a custom action (most likely the adapted `FormValidationAction`). Potentially it can also be set by a custom policy, although I'm not too sure yet how the default action would access that 🤔